po/cs.po

```diff
@@ -218,162 +225,168 @@ msgstr "Nová skupina"
 #: src/pages/SetPassword.vue:13
 #, fuzzy
 msgid "New password"
-msgstr "Nová zpráva"
+msgstr "Nové heslo"
 
 #: src/pages/Register.vue:17
 msgid "Next"
-msgstr ""
+msgstr "Další"
 
 #: src/pages/ChatList.vue:46
 msgid "No chats available"
-msgstr ""
+msgstr "Nenalezena žádná konverzace"
 
 #: src/pages/DeviceList.vue:14
 #, fuzzy
 msgid "No linked devices"
-msgstr "Obnovit"
+msgstr "Žádná připojená zařízení"
 
 #: src/pages/MessageList.vue:25
 msgid "No Messages available."
-msgstr ""
+msgstr "Nenalezeny žádné zprávy."
 
 #: src/components/Message.vue:46 src/components/Message.vue:72
 msgid "Not supported mime type:"
-msgstr ""
+msgstr "Nepodporovaný typ mime:"
 
 #: src/pages/EditGroup.vue:8 src/pages/NewGroup.vue:8
 msgid "Note, you can't add yourself to a group."
-msgstr ""
+msgstr "Pozor, není možné přidat sám/samu sebe do skupiny."
 
 #: src/pages/About.vue:13
 msgid "or by becoming a"
-msgstr ""
+msgstr "nebo jako"
 
 #: src/pages/Verification.vue:4
 msgid "or disable it on Android/IOs"
-msgstr ""
+msgstr "nebo deaktivovat pin na Androidu/IOs"
 
 #: src/pages/Password.vue:2
 msgid "Password is wrong"
-msgstr ""
+msgstr "Nesprávné heslo"
 
 #: src/pages/SetPassword.vue:4
 msgid "Passwords don't match"
-msgstr ""
+msgstr "Rozdílná hesla"
 
 #: src/components/AddContactModal.vue:17 src/components/EditContactModal.vue:15
 msgid "Phone"
-msgstr ""
+msgstr "Telefon"
 
 #: src/components/AttachmentBar.vue:11
 msgid "Photo"
-msgstr ""
+msgstr "Foto"
+
+#: src/components/StartChatModal.vue:11
+msgid ""
+"New conversations are temporarily disabled. You can create a new chat from "
+"the linked Desktop Client. Alternatively, your counterparty could start the "
+"discussion."
+msgstr "Prosím požádejte vašeho partnera, aby začal konverzaci jako první."
```

Comment:
One last thing before the merge, we changed the wording here, maybe you want to update the translation.

Reply:
Shall be resolved now. And, the message was present twice - I have deleted one of them.